Observations/obs.md

```diff
@@ -0,0 +1,152 @@
+
+Most of the information you store in OpenMRS is in the form of Observations, and most Observations happen in an Encounter. When you enter a form in OpenMRS, typically one Encounter is created with anywhere between tens or hundreds of Observations.
+
+Note that an individual Observation is valid only at one moment in time, and it does not carry forward. You may query the system for the last observation for pregnancy status but this does not tell you whether or not the patient is pregnant at any point after the moment of that observation.
+
+Examples of observations include Serum Creatinine of 0.9mg/dL or Review of cardiopulmonary system is normal.
+ 
+## Available operations. 
+
+1. [List observations](#list-observations)
+2. [Create an observation](#create-an-observation)
+3. [Update an observation](#update-an-observation)
+4. [Delete an observation](#delete-an-observation)
+
+### List observations
+
+* #### List all observations.
+    
+    Quickly filter observations with given query parameters. 
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+
+    #### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *patient* | `target_patient_uuid` | patient resource UUID
+    *concept* | `target_concept_uuid`| concept resource UUID (this parameter to be used with patient)
+
+    ```console
+    GET /obs?patient=070f0120-0283-4858-885d-a20d967729cf"
+     ```
+    
+* #### Query observations by UUID.
+
+    Retrieve an observation by its UUID.
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+    
+    ```console
+    GET /obs/:target_observation_uuid
+    ```
+   
+### Create an observation
+
+* To Create an observation you need to specify below attributes in the request body. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *person* | `Person UUID` | the Person this Obs is acting on.
+    *obsDateTime* | `String` | the time this Obs took place.
+    *concept* | `String` | the coded value/name given to an obs when it is made.
+    *location* | `String` | the location this Obs took place (was taken).
+    *order* | `String` | the order of an Obs.
+    *encounter* | `String` | what obs are collected and grouped together into. An encounter is a visit.
+    *accessionNumber* | `String` | a unique identifier assigned to each Obs.
+    *groupMembers* | `Array[]: Obs` |  a list of Obs grouped under this Obs
+    *comment* | `String` | the comment written in an Obs.
+    *value* | `String` | various values saved from a given Obs.
+    *status* | ENUM | the status of the observation
```

Comment:
[Here](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Obs.java#L103-L105) is the list of possible status values.

Please make the type "String" and use this description:

```
`PRELIMINARY`, `FINAL`, `AMENDED`
```